src/routes/EventsPage/index.js

```diff
@@ -1,32 +1,41 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import { connect } from 'react-redux'
+
 import RichLayout from 'components/RichLayout'
-import { EventsFetcher } from 'endpoints'
 import Events from '../../components/Content/Events'
+import EventModel from 'endpoints/models/EventModel'
+import EVENTS_ENDPOINT from 'endpoints/events'
+import withFetcher from '../../endpoints/withFetcher'
+
+class ContentWrapper extends React.Component {
+  static propTypes = {
+    events: PropTypes.arrayOf(PropTypes.instanceOf(EventModel)).isRequired // From withFetcher
+  }
+
+  render () {
+    return <Events events={this.props.events}/>
+  }
+}
+
+const FetchingContentWrapper = withFetcher(EVENTS_ENDPOINT)(ContentWrapper)
 
 class EventsPage extends React.Component {
   static propTypes = {
-    language: PropTypes.string.isRequired,
     location: PropTypes.string.isRequired
   }
 
   render () {
     return (
       <RichLayout location={this.props.location}>
-        <EventsFetcher>
-          <Events />
-        </EventsFetcher>
+        <FetchingContentWrapper/>
       </RichLayout>
     )
   }
 }
 
 function mapStateToProps (state) {
-  return {
-    language: state.router.params.language,
-    location: state.router.params.location
-  }
+  return {location: state.router.params.location}
```

Comment:
We don't need location in the DisclaimerPage props, RichLayout is connected to the state itself.
So we disconnect the EventsPage from the state 🎉

Reply:
done